app/models/notifications/queries/MucrInfo.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.notifications.queries
+
+import models.notifications.EntryStatus
+import play.api.libs.json.Json
+
+case class MucrInfo(
+  ucr: String,
+  parentMucr: Option[String] = None,
+  entryStatus: Option[EntryStatus] = None,
+  isShut: Option[Boolean] = None,
+  movements: Seq[MovementInfo] = Seq.empty
```

Comment:
Actually, doesn't it need to `override` ?

Reply:
Apparently not.